.gitignore.txt

```diff
@@ -0,0 +1,3 @@
+node_modules/
+test.md
```

Comment:
- [ ]  Kindly remember to add a test.md file with the text "This file should be ignored by git" to your repository (This is the file you should ignore on .gitignore file `test.md/` )